service/history/historyEngine.go

```diff
@@ -1437,7 +1438,7 @@ func (e *historyEngineImpl) RespondActivityTaskCompleted(
 		scope := e.metricsClient.Scope(metrics.HistoryRespondActivityTaskCompletedScope).
 			Tagged(
 				metrics.NamespaceTag(namespace),
-				metrics.WorkflowTypeTag(token.WorkflowType),
+				metrics.WorkflowTypeTag(workflowTypeName),
```

Comment:
i believe you can remove this metrics tag

Reply:
I synced with Max and we assumed it's better to keep the tag. On top of it, removing it will require cleanup on prometheus side due to change of tags on metric.